spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,136 @@
+'use strict'
+
 describe("Gilded Rose", function() {
 
-  it("should do something", function() {
-    update_quality();
-  });
+  beforeEach(() => {
+    items = []
+    // items.push(new Item('+5 Dexterity Vest', 10, 20));
```

Comment:
When refactoring, remove code rather than comment it out. If the code in the file is not in the source code repository (git), you can commit your code first, then refactor.